ui/raidboss/data/03-hw/alliance/weeping_city.txt

```diff
@@ -0,0 +1,416 @@
+### THE WEEPING CITY OF MHACH
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0 "--Reset--" sync /00:0839:.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~~~~~~~#
+# ARACHNE EVE #
+#~~~~~~~~~~~~~#
+
+# -ii 368 1835 1836
+# -ic "Webmaiden" "Spitting Spider" "Skittering Spider" "Earth Aether"
+
+0 "--sync--" sync /00:0839:.*Queen's Room will be sealed off/
+9.8 "Dark Spike" sync /:Arachne Eve:1823:/
+21.6 "Silken Spray" sync /:Arachne Eve:1824:/
+30.4 "Arachne Web x3" duration 5 # sync /:Arachne Eve:185E:/
+36.8 "--untargetable--"
+41.9 "Tremblor x3" sync /:Arachne Eve:1837:/ # 3 separate abilities, 1837/1836/1835
+48.4 "--sync--" sync /:Arachne Eve:1889:/
+49.9 "Pitfall" sync /:Arachne Eve:1825:/
+51.9 "--targetable--"
+52.1 "--sync--" sync /:Arachne Eve:1847:/ # Sticky Wicket head markers
+55.7 "Sticky Wicket x3" # sync /:Arachne Eve:183F:/
+64.1 "Silken Spray" sync /:Arachne Eve:1824:/
+
+# Intermission up in her web. There's a LOT of stuff going on here,
+# but the add spawns and abilities are dependent on what the team kills.
+# Implosion is the only real consistency we have through the intermission.
+95.7 "Implosion" sync /:Arachne Eve:1833:/ window 95.7,2.5
+113.9 "Implosion" sync /:Arachne Eve:1833:/
+132.1 "Implosion" sync /:Arachne Eve:1833:/
+150.3 "Implosion" sync /:Arachne Eve:1833:/ jump 95.7
+168.5 "Implosion"
+186.7 "Implosion"
+
+# Back on the ground for a relatively short bridge block
+200.0 "Dark Spike" sync /:Arachne Eve:1823:/ window 100,5
```

Comment:
Is there a starts casting for this to start this sync earlier?

Reply:
So there is, good point. It's only 2.7 seconds earlier, but a gain is a gain.